api/base/settings/defaults.py

```diff
@@ -167,6 +167,9 @@
 
 DEBUG_TRANSACTIONS = DEBUG
 
+JWT_SECRET = 'osf_api_cas_login_jwt_secret_32b'
+JWE_SECRET = 'osf_api_cas_login_jwe_secret_32b'
```

Comment:
These settings should not be allowed to be set to the default value. You should add validation like we have in `website/settings/__init__.py` to ensure this are changed in `local.py`.